packages/typespec-ts/test/modularIntegration/generated/azure/example/basic/.gitignore

```diff
@@ -0,0 +1,6 @@
+/**
```

Comment:
why we need to have this file inside the generated folder?

Reply:
this is auto-generated - https://github.com/Azure/autorest.typescript/blob/main/packages/typespec-ts/test/commands/run.ts#L87